Dumper/TranslationDumper.php

```diff
@@ -194,6 +211,41 @@ private function dumpTranslations($route, array $formats, $target)
         }
     }
 
+    private function dumpMergedTranslations($route, array $formats, $target)
+    {
+        foreach ($this->getTranslations() as $locale => $domains) {
+            foreach ($formats as $format) {
+                $content = $this->engine->render(
+                    'BazingaJsTranslationBundle::getTranslations.' . $format . '.twig',
+                    array(
+                        'translations' => array($locale => $domains),
+                        'include_config' => false,
+                    )
+                );
+
+                $file = sprintf(
+                    '%s/%s',
+                    $target,
+                    strtr(
+                        $route->getPath(),
+                        array(
+                            '{domain}' => sprintf('%s/%s', $locale, $locale),
```

Comment:
Sorry @bensampaio. What I wanted to say was that we should remove the duplication of folders like @stof was saying.

I don't see a clear advantage on using folders here, specially duplicating the name of the file. We could use folders if the name didn't have the locale. But in this case it has, so we can remove that duplication.

Sorry for my mistake.

What do you think?